### PR TITLE
Upgrade dependencies + set up Travis

### PR DESCRIPTION
**Pull request by gbalabasquer**

### CODE REVIEW

File: src/test/DssAction.t.sol [hunk 1]
```diff
@@ -78,7 +78,7 @@ contract ActionTest is DSTest {
     FlipperMom flipperMom;
     DssAutoLine autoLine;
 
-    ChainLog log;
+    ChainLog clog;
```
Comment (brianmcmichael) [inline src/test/DssAction.t.sol:81]: lol

**Review by brianmcmichael — CHANGES_REQUESTED** [review]
Getting an error on build here. Looks like remappings. Did you need to commit a config file?

File: src/test/DssAction.t.sol (hunk 1 above)
Comment (gbalabasquer) [reply src/test/DssAction.t.sol:81]: :)

**Comment by gbalabasquer** [comment]
> Getting an error on build here. Looks like remappings. Did you need to commit a config file?

Hmmm no, it doesn't need a config file. Which is the exact error?

**Comment by gbalabasquer** [comment]
The config.json was added in another branch, which might be an issue now. I'll have to rebase and see how that affects the new dependencies.

**Comment by brianmcmichael** [comment]
Tests pass locally. Remapping issue because `osm` dep was removed. `rm -rf lib/osm` did the trick.

**Pull request merged**